packages/core/src/components/ModalNew/Modal/Modal.tsx

```diff
@@ -32,13 +35,18 @@ const Modal = forwardRef(
       onClose = () => {},
       anchorElementRef,
       alertModal,
+      container = document.body,
```

Comment:
I don't think the name is good enough, when you think about it the modal is not under a certain container but it takes the whole page

Reply:
I followed MUI's convention (they use container), also we use the same convention in our Dialog and Tooltip components..
what do you suggest?

Reply:
portalTarget, but you make a good point about being consistent, so let's keep it this way